vaadin-cdi/src/test/java/com/vaadin/cdi/CdiVaadinServletTest.java

```diff
@@ -61,6 +66,14 @@ public void setUp() throws ServletException {
         Mockito.when(servletContext.getAttribute(Lookup.class.getName()))
                 .thenReturn(lookup);
 
+        final DefaultApplicationConfigurationFactory applicationConfigurationFactory
```

Comment:
There is no need to mock `DefaultApplicationConfigurationFactory` or its interface.
It's enough to properly mock `ServletContext`:
the `ApplicationConfiguration`  is retrieved from the context by the attribute `ApplicationConfiguration.class.getName()`.

If there is no attribute value then the factory is called. But if the attribute is given right away then it will be just returned without asking the factory.

But this is not really important. It just allows to reduce a number of lines.